docs/reviewing-release-prs.md

```diff
@@ -0,0 +1,176 @@
+# Reviewing Release PRs
+
+In order to publish a new release of the monorepo, a release PR must be created. But before a release PR is merged, it is absolutely critical — even if you are the author of the PR — to review the changes that are about to be published and to ensure that consumers know about them effectively.
+
+This document shows you how to do that.
+
+## 0. tl;dr — if you've done this sort of thing before
+
+- [ ] Are there any packages with version bumps that should not be published?
+- For all packages being published:
+  - [ ] Does the new version string sufficiently communicate the impact of changes?
+  - [ ] Are there any changelog entries that do not need to be listed?
+  - [ ] Are there any changelog entries that are placed in the wrong category (including "Unreleased")?
+  - [ ] Do all changelog entries explain the changes in a clear and straightforward manner and detail exact changes to the package's interface?
+  - [ ] Do all changes made through a pull request link to that pull request?
```

Comment:
Suggested re-wording:

```suggestion
  - [ ] Do all change entries include a link to the pull requests where that change was made?
```

"Changes made through a pull request" seemed like a circuitous way of referring to all changes outside of the RC. It seems fine to me to just say "change entries", and let the author figure out whether this applies.

Better to not imply that each change entry corresponds to just a single PR as well. We can have entries that relate to multiple PRs.